CONTRIBUTING.md

```diff
@@ -168,36 +165,41 @@ This tag is for boolean properties associated with the test.
 
 Some tests require the use of language features that are not directly described by the test file's location in the directory structure. These features should be specified with this tag. See the `features.txt` file for a complete list of available values.
 
+#### es5id
+**es5id**: [es5-test-id]
+
+**Deprecated.**
+
+This tag identifies the section number from the portion of the ECMAScript 5.1 or ECMAScript 3 standard that is tested by this test. It was automatically generated for tests that were originally written for the ES5 (or earlier) version of the test suite and are now part of the ES6 version. You can use the es5id to discover the relevant portion of the ECMAScript standard by looking up the section number in previous publications of the specification:
```

Comment:
remove the `or ECMAScript 3`

Reply:
@leobalter it's actually totally accurate.

Reply:
Might be useful to link to the report https://github.com/tc39/test262/wiki/Test262-Technical-Rationale-Report,-October-2017#specification-reference-ids